test/test-TabTracker.js

```diff
@@ -143,6 +143,10 @@ exports.test_TabTracker_reactivating = function*(assert) {
   for (let i in pingData) {
     let ping = pingData[i];
     assert.equal(ping.unload_reason, unloadReasons[i], "Unloaded for the expected reason");
+    // the page may or may not had enough time to be completely loaded, hence load_latency may
```

Comment:
The comment should read that the page will definitely not have the time to load. Otherwise we'll have an intermittent failure here.

Perhaps we need to load an artificial HTML file to guarantee this won't happen.